ui/packages/ui/src/app/pages/createConnector/connectorSteps/ConfigureConnectorTypeForm.tsx

```diff
@@ -34,14 +34,7 @@ export const ConfigureConnectorTypeForm: React.FunctionComponent<IConfigureConne
   const basicValidationSchema = {};
 
   const formatPropertyDefinitions = (propertyValues: ConnectorProperty[]) => {
-    const propertyValuesCopy = JSON.parse(JSON.stringify(propertyValues));
```

Comment:
Somehow this operation ends up modifying props(basicPropertyDefinitions in configureConnectorTypeForm) and state(in CreateConnectorPage) value direclty.  

i am adding "JSON.parse(JSON.stringify(propertyValues))" to break the reference else it may break the handling in other places.